buildSrc/src/main/groovy/com/uber/okbuck/experimental/bazel/AndroidLibraryRuleComposer.groovy

```diff
@@ -0,0 +1,28 @@
+package com.uber.okbuck.experimental.bazel
+
+import com.uber.okbuck.composer.AndroidBuckRuleComposer
+import com.uber.okbuck.core.model.AndroidLibTarget
+
+final class AndroidLibraryRuleComposer extends AndroidBuckRuleComposer {
```

Comment:
Ah, I just wrote out a long reply about this and then I think github went down for 2 minutes.

The gist of it was, I've added a commit that introduces BazelRuleComposer. It's a trait as opposed to a subclass of BuckRuleComposer because I need to extend both AndroidBuckRuleComposer and JavaBuckRuleComposer, which each extend BuckRuleComposer.